platform/pulp/platform/models/content.py

```diff
@@ -76,8 +76,11 @@ class Artifact(Model):
     :cvar downloaded: The associated file has been successfully downloaded.
     :type downloaded: BooleanField
 
-    :cvar published_path: The relative published path.
-    :type published_path: models.FileField
+    :cvar relative_path: The path to publish this artifact to relative to
+                         the :class:`Content` it is associated with; if the
+                         :class:`Content` has a single :class:`Artifact`
+                         this should simply be the filename.
```

Comment:
I think this would be clearer.
"""
The artifact's path relative to the associated :class:`Content`.
This path is incorporated in the absolute storage path of the `file` and its
published path relative to the root publishing directory.  At a minimum the path
will contain the file *name* but may also include sub-directories.
"""